foundation/src/main/java/org/researchstack/foundation/core/models/result/StepResult.java

```diff
@@ -27,7 +29,10 @@
 
     private Map<String, T> results;
 
-    private AnswerFormat answerFormat;
+    private Step step;
+
+    //TODO: See if we can remove this... What if we remove this for foundation, but add it for backwards compat foundation
+//    private AnswerFormat answerFormat;
```

Comment:
Please sign/date TODO comments.

Reply:
Removed TODO. This is already marked as deprecated and we'll remove this method when moving to 2.0